Kickstarter-iOS/Features/BackerDashboardPages/SwiftUI/BackerDashboardTabView.swift

```diff
@@ -0,0 +1,65 @@
+import Library
+import SwiftUI
+
+@available(iOS 15.0, *)
+struct BackerDashboardTabBarView: View {
+  @State var currentTab: BackerDashboardTab = .backed
+  @State var backedProjects: Int
+  @State var savedProjects: Int
+
+  @Namespace var animation
+
+  var selectTab: (BackerDashboardTab) -> ()
+
+  var body: some View {
+    ScrollView(.horizontal, showsIndicators: false) {
+      HStack(spacing: 48) {
+        TabView(tab: .backed, number: backedProjects)
+        TabView(tab: .saved, number: savedProjects)
+      }
+      .padding(.leading, 21)
+    }
+  }
+
+  // MARK: @ViewBuilders
+
+  private func TabView(tab: BackerDashboardTab, number: Int) -> some View {
+    Button {
+      currentTab = tab
```

Comment:
In the same way, in the long run, I don't think we should update the currentTab here. Instead, we tell the model that a new tab was selected, and then we update our UI only once the model tells us a different tab is active.